webapp/components/survey/CategoryDetails/store/actions/category/useUpdateCategoryProp.js

```diff
@@ -32,7 +28,12 @@ export const useUpdateCategoryProp = ({ setState }) => {
       const category = State.getCategory(statePrev)
       const categoryUuid = Category.getUuid(category)
 
-      _putCategoryProp({ dispatch, surveyId, categoryUuid, key, value, setState })
+      dispatch(
+        debounceAction(
+          _putCategoryProp({ surveyId, categoryUuid, key, value, setState }),
+          `category_prop_update_${categoryUuid}`
+        )
+      )
```

Comment:
how is the time of response for the user? because maybe you can do an optimistic update, updating the state outside of the debounce action